extension/lib/time_tracker_view.js

```diff
@@ -88,7 +88,7 @@ class TimeTrackerView {
 
         switch (this.time_remaining_granularity) {
             case timeRemainingGranularity.MINUTE:
-                return `${Math.ceil(time_remaining_m)}m`;
+                return `${Math.ceil(time_remaining_m)}`;
```

Comment:
This resulted in the M being big; I reverted it to what we had before, with the number big and afterward "min remaining". I changed back to min bc when I saw m I was a little confused initially, and I didn't recall the old style being too bulky. Let me know if you prefer this though! Wasn't sure if intentional or accidental